tools/nni_cmd/nnictl_utils.py

```diff
@@ -373,6 +379,146 @@ def webui_url(args):
+    full_path = '/' + '/'.join([user_name, 'nni', 'experiments', nni_config.get_config('experimentId')])
+    print_normal('deleting {0} in hdfs'.format(full_path))
+    hdfs_client.delete(full_path, recursive=True)
+    output_dir = nni_config.get_config('experimentConfig').get('paiConfig').get('outputDir')
+    if output_dir:
+        pattern = re.compile('hdfs://(?P<host>([0-9]{1,3}.){3}[0-9]{1,3})(:[0-9]{2,5})?(?P<baseDir>/.*)?')
+        match_result = pattern.match(output_dir)
+        if match_result:
+            output_host = match_result.group('host')
+            output_directory = match_result.group('baseDir')
+            if output_host == host:
+                print_normal('deleting {0} in hdfs'.format(output_directory))
+                hdfs_client.delete(output_directory, recursive=True)
+
+def experiment_clean(args):
+    '''clean up the experiment data'''
+    nni_config = Config(get_config_filename(args))
+    while True:
+        print('INFO: clean up all data for experiment {0}'.format(args.id))
+        inputs = input('INFO: do you want to continue?[y/n]:')
+        if not inputs.lower() or inputs.lower() in ['n', 'no']:
+            print_normal('Exit!')
+            exit(0)
+        elif inputs.lower() not in ['y', 'n', 'yes', 'no']:
+            print_warning('please input Y or N!')
+        else:
+            break
+    #clean local data
+    home = str(Path.home())
+    local_dir = os.path.join(home, 'nni', 'experiments', str(args.id))
+    local_clean(local_dir)
+    platform = nni_config.get_config('experimentConfig').get('trainingServicePlatform')
+    if platform == 'remote':
+        remote_clean(nni_config)
+    elif platform == 'pai':
+        hdfs_clean(nni_config)
+    elif platform != 'local':
+        #TODO: support all platforms
+        print_warning('platform {0} clean up not supported yet!'.format(platform))
+        exit(0)
+    experiment_config = Experiments()
+    experiment_config.remove_experiment(args.id)
+    print_normal('Success!') 
+
+def get_platform_dir(experiment_dict, platform):
+    '''get directories in platform'''
+    dir_list = []
+    for key in experiment_dict.keys():
+        if platform != experiment_dict[key]['platform']:
+            continue
+        file_name = experiment_dict[key].get('fileName')
+        nni_config = Config(file_name)
+        if platform == 'remote':
+            machine_list = nni_config.get_config('experimentConfig').get('machineList')
+            for machine in machine_list:
+                host = machine.get('ip')
+                port = machine.get('port')
+                remote_dir = '/' + '/'.join(['tmp', 'nni', 'experiments', nni_config.get_config('experimentId')])
+                dir_list.append(host + ':' + str(port) + remote_dir)
+        elif platform == 'pai':	
+            user_name = nni_config.get_config('experimentConfig').get('paiConfig').get('userName')	
```

Comment:
do you need to check whether this pai config is the same as the command arg config?

Reply:
no, this function will get all of directory in the related platform, the args config only have one directory.